src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventControllerDeleteEventMappingTest.java

```diff
@@ -0,0 +1,140 @@
+package uk.gov.hmcts.darts.event.controller;
+
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import uk.gov.hmcts.darts.common.entity.EventEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.testutils.GivenBuilder;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+import java.time.LocalDateTime;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.SUPER_ADMIN;
+import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.TRANSLATION_QA;
+
+@AutoConfigureMockMvc
+class EventControllerDeleteEventMappingTest extends IntegrationBase {
+
+    private static final String EVENT_MAPPINGS_ENDPOINT = "/admin/event-mappings/{event_handler_id}";
+
+    @Autowired
+    private transient MockMvc mockMvc;
+
+    @Autowired
+    private GivenBuilder given;
+
+    @Test
+    void deleteOk() throws Exception {
+        given.anAuthenticatedUserWithGlobalAccessAndRole(SUPER_ADMIN);
+        var entity = dartsDatabase.createEventHandlerData();
+        MockHttpServletRequestBuilder requestBuilder = delete(EVENT_MAPPINGS_ENDPOINT, entity.getId());
+        mockMvc.perform(requestBuilder).andExpect(status().isOk());
+    }
+
+    @Test
+    void wrongPermission() throws Exception {
```

Comment:
Could use pattern here to test all permissions that shouldn't have access: https://github.com/hmcts/darts-api/blob/master/src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventControllerGetEventMappingByIdTest.java#L64-L65

Reply:
done